lib/src/rules/avoid_annotating_with_dynamic.dart

```diff
@@ -59,7 +60,15 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   @override
   void visitSimpleFormalParameter(SimpleFormalParameter node) {
-    var type = node.type;
+    _checkNode(node, node.type);
+  }
+
+  @override
+  void visitSuperFormalParameter(SuperFormalParameter node) {
```

Comment:
It appears that we're not visiting `FieldFormalParameter`s. I think we should either include those in the lint or not include `SuperForrmalParameter`s (probably the former). Unfortunately, visiting `FieldFormalParameter`s is a breaking change.

Reply:
I wondered about that too and after sitting on it agree that we should widen current support to include `FieldFormalParameter`s.  It's breaking but I think likely to impact few folks in practice (and needless to say is I think what they would *want* the lint to do).

Reply:
I'll land this change as is and follow-up w/ one to add `FieldFormalParameter`s  -- in case we ever want to revert just that.